cmd/evm/main.go

```diff
@@ -228,26 +228,27 @@ type ruleSet struct{}
 
 func (ruleSet) IsHomestead(*big.Int) bool { return true }
 
-func (ruleSet) IsAtlantis(*big.Int) bool {
-	// Default true for tests
-	return true
-}
+//set IsAtlantis to true by default for tests
+func (ruleSet) IsAtlantis(*big.Int) bool { return true }
 
-func (ruleSet) IsAgharta(*big.Int) bool {
-	return true
-}
+//set IsAgharta to true by default for tests
+func (ruleSet) IsAgharta(*big.Int) bool { return true }
 
 func (ruleSet) GasTable(*big.Int) *vm.GasTable {
+	//IsAgharta will always return true here,
+	//just have gastable default to returning the Agharta GasTable
 	return &vm.GasTable{
 		ExtcodeSize:     big.NewInt(700),
 		ExtcodeCopy:     big.NewInt(700),
+		ExtcodeHash:     big.NewInt(400),
 		Balance:         big.NewInt(400),
 		SLoad:           big.NewInt(200),
 		Calls:           big.NewInt(700),
 		Suicide:         big.NewInt(5000),
-		ExpByte:         big.NewInt(10),
+		ExpByte:         big.NewInt(50),
```

Comment:
Not sure why that was changed, but this is only used for tests so has no functional change, will update now anyway

Reply:
actually I will keep it like this because for this test ruleSet, isAgharta is defaulted to true so it makes sense to have the gastable Agharta specific, even though it isn't used